package.json

```diff
@@ -15,11 +15,16 @@
     "@nuxtjs/auth-next": "5.0.0-1643791578.532b3d6",
     "@nuxtjs/axios": "^5.13.6",
     "core-js": "^3.19.3",
+    "lodash": "^4.17.21",
+    "lodash.debounce": "^4.0.8",
```

Comment:
What is this?
It's not included in skyloud package? 🤔

Reply:
Nope